ledger/src/blockstore_db.rs

```diff
@@ -1138,6 +1148,15 @@ where
     pub fn put_bytes(&self, key: C::Index, value: &[u8]) -> Result<()> {
         self.backend.put_cf(self.handle(), &C::key(key), value)
     }
+
+    /// Retrieves the specified RocksDB integer property of the current
+    /// column family.
+    ///
+    /// Full list of properties that return int values could be find
+    /// [here](https://github.com/facebook/rocksdb/blob/08809f5e6cd9cc4bc3958dd4d59457ae78c76660/include/rocksdb/db.h#L654-L689).
+    pub fn get_int_property(&self, name: &str) -> Result<Option<u64>> {
+        self.backend.get_int_property_cf(self.handle(), name)
```

Comment:
@steviez: this API.

For instance, you can obtain the mem-table size by calling this API with "rocksdb.cur-size-all-mem-tables".

All the parameters are listed [here](https://github.com/facebook/rocksdb/blob/08809f5e6cd9cc4bc3958dd4d59457ae78c76660/include/rocksdb/db.h#L654-L689).  Not sure why rust-rocksdb doesn't define consts for these property keys.